pkg/volume/scaleio/sio_util.go

```diff
@@ -107,6 +109,12 @@ func validateConfigs(config map[string]string) error {
 	if config[confKey.system] == "" {
 		return systemNotProvidedErr
 	}
+	if config[confKey.storagePool] == "" {
```

Comment:
@jsafrane You are correct, existing/deployed config are ok, but newer configs will require these values to be provided in subsequent deployments.  I will add that to the release notes and the documentations will also be updated (separate PR).